.github/workflows/ci.yml

```diff
@@ -65,64 +65,7 @@ jobs:
     # to find the list of authors and replace them, so any changes to the
     # formatting of the contains(fromJSON()) structure below will need to be
     # reflected in that workflow too.
-    runs-on: >-
```

Comment:
Finally we will be able to get rid of this list in yaml (@ashb - ci-infra instruction fix will follow once we merge it).